backend/src/main/java/com/google/sps/notifiers/Notifier.java

```diff
@@ -0,0 +1,18 @@
+// Class used to send users emails regarding their status in regards to their Ad-Lib Meeting.
+
+import java.lang.*;
+import java.io.File;
+import com.sendgrid.*;
+
+
+public class Notifier {
+
+    // Member Variables 
+    
+    String Name;        /* String represenintg the user who is to be notified's name */
```

Comment:
nit: remove new line (line 11), one space after String Name instead of tab, "name of participant to be notified" instead of current comment, private String?

Reply:
Formatted